common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -109,6 +109,11 @@ describe 'Problem', ->
       @bind = @problem.bind
       spyOn @problem, 'bind'
 
+    describe 'with reset error', ->
+      it 'error message div is visible', ->
+        @problem.render('', 'foo');
+        expect(@problem.el.find('.reset-error').css("display")).toBe('inline')
```

Comment:
@mehrozezahid We also need to add test when there is not error received from server.